### ISSUE TITLE
[BUG] Dropdown cut-off inside a modal.

### ISSUE THREAD

**Issue by tspringborg** [kirbydesign/designsystem#2433]
## Describe the bug
Inside a modal, kirby dropdown is cut off... but only under certain conditions. Most importantly there should be content in the modal and the position of the dropdown button has to be in a very specific place. (As evident from the video)
Furthermore, if you are testing this issue in a browser, you should emulate an iphone SE or a similar small device.

## Describe how to reproduce the bug
I've made a repo reproducing the bug. https://github.com/tspringborg/kirby-playground/tree/dropdown-issue
Remember to emulate an iPhone SE (or similar device dimensions which will probably also work)

## Which Kirby version was used?
6.3.1

## What was the expected behavior?
Should be able to pick any dropdown item. Regardless of whether the dropdown opens up or down.

## Add any screenshots

Image displaying how the top part of the dropdown gets cut off:
![image](https://user-images.githubusercontent.com/1595605/186146247-478a60f2-1955-4b17-99f7-b0fcb48a8330.png)

Video displaying the cutoff in action:
https://user-images.githubusercontent.com/1595605/186145702-27ee501f-300d-4fa0-a2d5-3df84de30ed1.mov

## Please complete the following information:
- OS: MacOS
- Chrome
- Version 104.0.5112.79 (Official Build) (arm64)

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not the intended behavior; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
